### PR TITLE
Maintace/code refactor

### PR DESCRIPTION
**Pull request by wladyslawmajdanik**
- Code Refactor
- Simplified Structures
-  CleanUp

### CODE REVIEW

File: src/RootStackNavigator.tsx [hunk 1]
```diff
@@ -10,12 +10,12 @@ import * as Types from './types';
 
 const Stack = createStackNavigator<Types.RootStackParams>();
 
-export const RootStackScreen: React.FC = () => {
-  const isLoggedIn = useSelector(selectors.isLoggedIn);
+export const RootStackNavigator: React.FC = () => {
+  const user = useSelector(selectors.getUser);
```
Comment (bklukaczewski) [inline src/RootStackNavigator.tsx]: you could also keep `isLoggedIn` and have:
```
const isLoggedIn = !!useSelector(selectors.getUser);
```

**Review by bklukaczewski — APPROVED** [review]
Amazing, each line of code less is better in our case 💪

File: src/components/Button/index.tsx [hunk 2]
```diff
@@ -19,15 +18,14 @@ export const LogInButton: React.FC<Props> = ({ action, title }) => {
 
 const styles = StyleSheet.create({
   button: {
-    width: 200,
-    height: 40,
-    margin: 10,
-    padding: 10,
+    width: '50%',
```
Comment (bklukaczewski) [inline src/components/Button/index.tsx]: maybe let's use flex here? We don't really use percentage based width much in RN

File: src/screens/SignInScreen/index.tsx [hunk 3]
```diff
@@ -65,10 +64,9 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
   },
   input: {
-    width: 200,
-    height: 40,
-    margin: 10,
-    padding: 10,
+    width: '50%',
```
Comment (bklukaczewski) [inline src/screens/SignInScreen/index.tsx]: same here, maybe just `flex: 1`?

File: src/store/actions/appAction.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+export enum AppAction {
```
Comment (bklukaczewski) [inline src/store/actions/appAction.ts]: it could also be in the `actions` file

**Review by bklukaczewski — APPROVED** [review]
amazing to remove so much code. it should be much more approachable by junior developers :)

**Review by wojtekstx — APPROVED** [review]
Good job! Please provide @bklukaczewski suggestions

**Pull request merged**